### PR TITLE
DOC-1128: RC - Doc Active-Active workflow changes

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1128/rc/subscriptions/create-active-active-subscription/).

File: content/rc/subscriptions/create-active-active-subscription.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+- [Define the regions](#define-regions) for each database instance
+- Define unique [CIDR addresses](#define-cidr-addresses) for each instance
+- [Define throughput](#define-throughput) (read and write operations) for each region
+
+Active-Active databases consist of multiple copies (also called _instances_) deployed to different regions throughout the world.
+
+This reduces latency for local users and improves availability should a region fail.
+
+Consistency between each instance is maintained in the background; that is, each copy eventually includes updates from every region.  As a result, [memory size]({{<relref "/rc/databases/create-database#memory-size">}}) and throughout increase.
```
Comment (kaitlynmichael) [inline content/rc/subscriptions/create-active-active-subscription.md:23]:
```suggestion
Consistency between each instance is maintained in the background; that is, each copy eventually includes updates from every region.  As a result, [memory size]({{<relref "/rc/databases/create-database#memory-size">}}) and throughput increase.
```

**Review by kaitlynmichael — APPROVED** [review]
Looks great!

File: content/rc/subscriptions/create-active-active-subscription.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/subscriptions/create-active-active-subscription.md]: Fixed.

**Pull request merged**